src/Store/KeyBindingsStoreConnector.re

```diff
@@ -33,6 +33,16 @@ let start = () => {
         command: "list.selectBackground",
         condition: "quickmenuCursorEnd" |> parseExp,
       },
+      {
+        key: "<S-C-F>",
+        command: "workbench.action.findInFiles",
+        condition: "editorTextFocus" |> parseExp,
+      },
+      {
+        key: "<D-S-F>",
```

Comment:
I think this is how you specify `Cmd-Shift-F`?

Reply:
Correct! We need to have a nicer parser for keybindings though (something that can support both the VSCode style: Cmd+Shift+F and the Vim style: <D-S-F>)